src/main/frontend/db.cljs

```diff
@@ -781,6 +782,20 @@
            (map :page/name)
            distinct))))
 
+(defn get-published-pages
```

Comment:
We need to separate db, handlers and the view components.

Reply:
Queries that is in db.clj might be handled in the following refractor PR.

Reply:
Ok.